src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star">0</div>
+      <div class="stars__star">0</div>
+      <div class="stars__star">0</div>
+      <div class="stars__star">0</div>
+      <div class="stars__star">0</div>
```

Comment:
```suggestion
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
```
Remove it everywhere